scripts/importer.py

```diff
@@ -18,60 +18,173 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
-
-
 """Tool to import data from other browsers.
 
 Currently only importing bookmarks from Netscape Bookmark files is supported.
 """
 
-
 import argparse
 
+browser_default_input_format = {
+    'chromium': 'netscape',
+    'ie': 'netscape',
+    'firefox': 'netscape',
+    'seamonkey': 'netscape',
+    'palemoon': 'netscape'
+}
```

Comment:
It's definitely useless here, but with 'native' input format support for things like Chrome or Firefox the browser name would allow the user to specify the profile name rather than a direct path, for example 
`importer.py firefox default` 
instead of 
`importer.py firefox ~/.mozilla/firefox/uzp27j1h.default`